RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -472,7 +474,29 @@ void EcalUncalibRecHitWorkerMultiFit::run(const edm::Event& evt,
         }
         uncalibRecHit.setJitter(timerh);
         uncalibRecHit.setJitterError(0.);  // not computed with weights
-      } else {                             // no time method;
+
+      } else if (timealgo_ == crossCorrelationMethod) {
+        float startTime = -25;
+        float stopTime = 25;
+        EcalUncalibRecHitTimingCCAlgo computeCC(startTime, stopTime);
+
+        uncalibRecHit.setJitterError(0.);
+        float timeStep = 1;
+        float tempt = 0;
+
+        std::vector<double> amplitudes;
+        for (unsigned int ibx = 0; ibx < activeBX.size(); ++ibx)
+          amplitudes.push_back(uncalibRecHit.outOfTimeAmplitude(ibx));
+        // seedTime = 0;
+
+        tempt = computeCC.computeTimeCC(*itdg, amplitudes, aped, aGain, fullpulse, uncalibRecHit);
+
+        uncalibRecHit.setJitter(-tempt);
+        if (tempt > stopTime - timeStep || tempt < startTime + timeStep)
+          uncalibRecHit.setJitterError(-timeStep / 25);
```

Comment:
In the first case it is a parameter, not necessarily 25 ns. It could be +-15.
Not sure how to put this, maybe as parameter in the python? 
Thanks for the (-timeStep / 25), I missed it in the first round

Reply:
If it may be changed at some point make it a parameter in the python config, yes. If not make it a named constant.